templates/steering-display/src/main.cpp

```diff
@@ -9,17 +9,26 @@
 #include <Arduino.h>
 #include <SPI.h>
 
+#include "can.hpp"
+
 #include "EVE_target.h"
 #include "EVE_commands.h"
 #include "tft.h"
 
+#define CAN_OE 10
+
 void setup() {
     Serial2.begin(115200);
     pinMode(EVE_CS, OUTPUT);
     digitalWrite(EVE_CS, HIGH);
     pinMode(EVE_PDN, OUTPUT);
     digitalWrite(EVE_PDN, LOW);
 
+    pinMode(CAN_OE, OUTPUT);
+	digitalWrite(CAN_OE, LOW);
```

Comment:
Just a formatting note. Not sure if this is just tabbed weird on the web?

Reply:
probably messed up when copy pasted. TODO